### DISCUSSION TITLE
Should we use i18n instead of recreating files?

### DISCUSSION THREAD

**Discussion by tony19** [tony19/vite-docs-template#7] [Q&A] [answered]
I'm wondering if we can use an i18n lib to support translations instead of having to recreate every `docs` file from the main repo. Not sure which would be easier for contributors and maintainers. 🤔

For instance:
* [`@intlify/vite-plugin-vue-i18n`](https://www.npmjs.com/package/@intlify/vite-plugin-vue-i18n)
* [VuePress i18n](https://vuepress.vuejs.org/guide/i18n.html#site-level-i18n-config)

cc @mrcego @ShenQingchuan

**Comment by mrcego** [comment]
I'm not sure about it, because according my knowledge, it drops forks from docs, right? I mean, contributors will keep up to date an unique repo with i18n tool into it.

**Comment by tony19** [comment, marked as answer]
Actually, I don't think those i18n libs will work in VitePress. Forking the Markdown files per language (as we're already doing) seems like the right way to go.
